source/api/commands/check.md

```diff
@@ -85,16 +85,34 @@ cy.get('#saveUserName').check()
 
 ### Select the radio with the value of 'US'
 
+```html
+<form>
+  <input type="radio" id="us-canada" value="CA">
+  <label for="us-canada">Canada</label>
+  <input type="radio" id="us-country" value="US">
```

Comment:
I like how for Canada we have `id=us-canada` but for US it is imply `id=us-country`. Could just be `id=us-north` and `id=us` :)

Reply:
Derp. Will fix this.